bash scripts to modify network in xva's files (#71)
xcp-ng/xcp-ng-tests: README.md

      community.general.apk:
        name: util-linux efitools
```

# Bash scripts

>  * get_xva_bridge.sh: a script to get the bridge's value inside a xva file and the compression method used for this xva file.

Reviewer: ```suggestion
 * get_xva_bridge.sh: a script to get the XAPI bridge value from inside a xva file and the compression method used for this xva file.
```

```

> vates@vlaptop xva]$ ./get_xva_bridge.sh alpine-minimal-3.12.0.xva

Reviewer: ```suggestion
$ ./get_xva_bridge.sh alpine-minimal-3.12.0.xva
```

ova.xml
alpine-minimal-3.12.0.xva's bridge network is: xapi1 and its compression method is: tar.
```

>  * set_xva_bridge.sh: a script to modify the bridge's value inside a xva file and the compression method used for this xva file if wanted. The original xva file is save before modification.

Reviewer: ```suggestion
 * set_xva_bridge.sh: a script to modify the XAPI bridge value inside a xva file and the compression method used for this xva file if wanted. The original xva file is saved before modification.
```

```

> - Usage: ./set_xva_bridge.sh [XVA's_filename] compression[zstd|gzip] bridge_value[xenbr0|xapi[:9]|...]

Reviewer: I missed this one:
```suggestion
- Usage: /path/to/set_xva_bridge.sh [XVA_filename] compression[zstd|gzip] bridge_value[xenbr0|xapi[:9]|...]
```

- All options are mandatory.

> vates@vlaptop xva]$ ./set_xva_bridge.sh alpine-minimal-3.12.0.xva zstd xenbr0

Reviewer: ```suggestion
$ /path/to/set_xva_bridge.sh alpine-minimal-3.12.0.xva zstd xenbr0
```

      community.general.apk:
        name: util-linux efitools
```

# Bash scripts

 * get_xva_bridge.sh: a script to get the bridge's value inside a xva file and the compression's method used for this xva file.

```
vates@vlaptop xva]$ ./get_xva_bridge.sh alpine-minimal-3.12.0.xva
ova.xml
alpine-minimal-3.12.0.xva's bridge network is: xapi1 and its compression's method is: tar.
```

 * set_xva_bridge.sh: a script to modify the bridge's value inside a xva file and the compression's method used for this xva file if wanted. The original xva file is save before modification.

```
- Usage: ./set_xva_bridge.sh [XVA's_filename] compression[zstd|gzip] bridge_value[xenbr0|xapi[:9]|...]
- All options are mandatory.

vates@vlaptop xva]$ ./set_xva_bridge.sh alpine-minimal-3.12.0.xva zstd xenbr0

> 

Reviewer 2: Useless empty line.

> 
> ```

Reviewer 2: ```suggestion
```
```
